llvm-external-projects/iree-dialects/lib/Dialect/LinalgTransform/Passes/TransformInterpreterPassBase.cpp

```diff
@@ -15,6 +15,7 @@
 #include "llvm/Support/Debug.h"
 #include "llvm/Support/FileSystem.h"
 #include "llvm/Support/FormatVariadic.h"
+#include "llvm/Support/Mutex.h"
```

Comment:
I'd just drop all this and only allow debug in single threaded mode.
In the future all this will need to be more tightly integrated with IREE anyway and hopefully transform IR attached separately and not dropped on the floor by the nested pass pipeline.